packages/camera/camera/android/src/test/java/io/flutter/plugins/camera/CameraTest.java

```diff
@@ -773,6 +800,22 @@ public void resumePreview_shouldSendErrorEventOnCameraAccessException()
     verify(mockDartMessenger, times(1)).sendCameraErrorEvent(any());
   }
 
+  @Test
+  public void startBackgroundThread_shouldStartNewThread() {
+    camera.startBackgroundThread();
+
+    verify(mockHandlerThread, times(1)).start();
+    assertEquals(mockHandler, TestUtils.getPrivateField(camera, "backgroundHandler"));
+  }
+
+  @Test
+  public void startBackgroundThread_shouldNotStartNewThreadWhenAlreadyCreated() {
+    camera.startBackgroundThread();
+    camera.startBackgroundThread();
+
+    verify(mockHandlerThread, times(1)).start();
```

Comment:
verify(mockHandlerThread, times(0)).start();  first. before you call `camera.startBackgroundThread();`.  It verifies the state before `startBackgroundThread()` is called.